packages/zosfiles/src/methods/list/doc/IDsmListOptions.ts

```diff
@@ -32,16 +32,6 @@ export interface IDsmListOptions extends IZosFilesOptions {
      */
     maxConcurrentRequests?: number;
 
-    /**
-     * The indicator that we want to show less data sets or members
-     */
-    maxLength?: number;
-
-    /**
-     * An optional search parameter that specifies the first data set name to return in the response document
-     */
-    start?: string;
```

Comment:
will this be considered a breaking change?

Reply:
It shouldn't - these are now inherited by extending `IListOptions`, so the properties are still intact while keeping the same types 😋